packages/plugin-react/src/index.d.ts

```diff
@@ -1,12 +1,29 @@
 import { Plugin } from 'vite'
-import { ParserOptions } from '@babel/core'
-
-type PluginFactory = (options?: Options) => Plugin
-
-declare const createPlugin: PluginFactory & { preambleCode: string }
+import { TransformOptions, ParserOptions } from '@babel/core'
 
 export interface Options {
-  parserPlugins: ParserOptions['plugins']
+  /**
+   * Enable `react-refresh` integration. Vite disables this in prod env or build mode.
+   * @default true
+   */
+  fastRefresh?: boolean
```

Comment:
This option is for Rollup users. I made sure to keep this plugin compatible with Rollup, since it seemed to be designed that way in the first place.